docs/docs-beta/docs/guides/deploy/deployment-options/aws.md

```diff
@@ -0,0 +1,252 @@
+---
+title: "Deploying Dagster to Amazon Web Services"
+description: To deploy Dagster to AWS, EC2 or ECS can host the Dagster webserver, RDS can store runs and events, and S3 can act as an IO manager.
+sidebar_position: 50
+---
+
+This guide provides instructions for deploying Dagster on Amazon Web Services (AWS). You can use EC2 or ECS to host the Dagster webserver and the Dagster daemon, RDS to store runs and events, and S3 as an I/O manager to store op inputs and outputs.
+
+## Hosting Dagster on EC2
+
+To host Dagster on a bare VM or in Docker on EC2, see "[Running Dagster as a service](/guides/deploy/deployment-options/deploying-dagster-as-a-service).
+
+## Using RDS for run and event log storage
+
+You can use a hosted RDS PostgreSQL database for your Dagster run/events data by configuring your `dagster.yaml` file:
+
+```python file=/deploying/dagster-pg.yaml
+storage:
+  postgres:
+    postgres_db:
+      username: my_username
+      password: my_password
+      hostname: my_hostname
+      db_name: my_database
+      port: 5432
+```
+
+In this case, you'll want to ensure that:
+
+- You provide the right connection strings for your RDS instance
+- The node or container hosting the webserver is able to connect to RDS
+
+Be sure that this file is present and `DAGSTER_HOME` is set on the node where the webserver is running.
+
+:::note
+
+Using RDS for run and event log storage doesn't require that the webserver be running in the cloud. If you're connecting a local webserver instance to a remote RDS storage, verify that your local node is able to connect to RDS.
+
+:::
+
+## Deploying in ECS
+
+{/* TODO turn back into <CodeReferenceLink> once that's implemented */}
+:::tip
+
+You can find the code for this example on [GitHub](https://github.com/dagster-io/dagster/tree/1.9.8/examples/deploy_ecs).
```

Comment:
Any reason this Github link is pinned to `1.9.8`?

Reply:
In old docs, this link is generated by a `<CodeReferenceLink>` component, but that component isn't implemented in new docs. This is basically a placeholder link until we implement that component -- here's the relevant Linear ticket: https://linear.app/dagster-labs/issue/DOC-685/implement-codereferencelink-component

Reply:
This should probably just be https://github.com/dagster-io/dagster/tree/master/examples/deploy_ecs in the meantime, though; I can change that.